geopy/geocoders/googlev3.py

```diff
@@ -248,7 +264,6 @@ def geocode(
             url = self._get_signed_url(params)
         else:
             url = "?".join((self.api, urlencode(params)))
-
```

Comment:
Could you please roll back this change? The new extra newline at the end the file too.